2.1/build/rpmbuild/SOURCES/0001-Define-GOPATH-if-there-is-none.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/Makefile.am b/Makefile.am
```

Comment:
This entire file can be only one instead of one for centos and other for fedora.

You can replace the content of this with the content of: 

https://github.com/01org/cc-oci-runtime/pull/599